bitnami/rabbitmq/README.md

```diff
@@ -7,14 +7,16 @@ RabbitMQ is an open source general-purpose message broker that is designed for c
 [Overview of RabbitMQ](https://www.rabbitmq.com)
 
 Trademarks: This software listing is packaged by Bitnami. The respective trademarks mentioned in the offering are owned by the respective companies, and use of them does not imply any affiliation or endorsement.
-                           
+
 ## TL;DR
 
 ```bash
-$ helm repo add bitnami https://charts.bitnami.com/bitnami
-$ helm install my-release bitnami/rabbitmq
+$ helm repo add my-repo REPO
+$ helm install my-release my-repo/rabbitmq
 ```
 
+Remember to replace the `REPO` placeholder by the repository from where you would like to deploy this Helm chart.
```

Comment:
What do you think about pointing to [here](https://github.com/bitnami/charts/issues/10833) ?
Maybe we should move that page to a informative page about the index (or bitnami repo).